### PR TITLE
Partial mouse update

### PR DESCRIPTION
**Pull request by mrElnekave**
Takes the two first commits from mouse updated which are urgent proper fixes.

### CODE REVIEW

File: rubato/game.py [hunk 1]
```diff
@@ -117,14 +117,14 @@ def update(cls):  # test: skip
         for event in sdl2.ext.get_events():
             sdl2.SDL_PumpEvents()
             if event.type == sdl2.SDL_QUIT:
-                Radio.broadcast("EXIT")
+                Radio.broadcast(Events.ZOOM)
```
Comment (tinmarr) [inline rubato/game.py:120]: This should be Events.Exit

File: rubato/game.py [hunk 2]
```diff
@@ -155,6 +155,37 @@ def update(cls):  # test: skip
                     },
                 )
 
+            if event.type in (sdl2.SDL_MOUSEBUTTONDOWN, sdl2.SDL_MOUSEBUTTONUP):
+                mouse_button = None
+                if event.button.state == sdl2.SDL_BUTTON_LEFT:
+                    mouse_button = "mouse 1"
```
Comment (tinmarr) [inline rubato/game.py:161]: This naming doesn't match what you have in the docs

**Review by tinmarr — CHANGES_REQUESTED** [review]
Things in game.py need to be changed (see comments)

File: rubato/game.py (hunk 2 above)
Comment (mrElnekave) [reply rubato/game.py:161]: Lol fixed in later commits in the original branch.

**Pull request merged**